message_adapter/cumulus_message.py

```diff
@@ -41,7 +41,7 @@ def load_remote_event(event):
             remote_event = json.loads(data['Body'].read().decode('utf-8'))
             replacement_targets = parsed_json_path.find(event)
             if not replacement_targets or len(replacement_targets) != 1:
-                raise Exception(f'Remote event configuration target {target_json_path} invalid')
+                raise LookupError(f'Remote event configuration target {target_json_path} invalid')
```

Comment:
For all these changes to `Exception`, pylint was failing/complaining in circle ci about `Raising too general exception: Exception (broad-exception-raised)`